### ISSUE TITLE
Buy with no sell

### ISSUE THREAD

**Issue by smileyung** [xtr33me/Reinforcement_Learning_for_Stock_Prediction#4]
Hi, I have a problem:

If I buy some stock,
and I don't sell them,
I will get no reward, right?

**Comment by xtr33me** [comment]
Correct....its due to the fact that the current models reward is based on the overall profit made with the buy of shares and then the selling of those same shares.  Ofcourse we have the ability to make this reward whatever it is we want as long as it makes sense.
